src/main/java/org/folio/circulation/services/LostItemFeeRefundContext.java

```diff
@@ -33,17 +30,23 @@
   private final String staffUserId;
   private final String servicePointId;
   private final Loan loan;
-  private final Collection<Account> accounts;
-  private final LostItemPolicy lostItemPolicy;
 
   private Collection<Account> accountsNeedingRefunds() {
-    if (!lostItemPolicy.isRefundProcessingFeeWhenReturned()) {
-      return accounts.stream()
+    if (!getLostItemPolicy().isRefundProcessingFeeWhenReturned()) {
+      return loan.getAccounts().stream()
```

Comment:
Why does this use the accounts for a loan when there is a separate list of accounts defined for the context?

Reply:
I'm using accounts from loan now and removed the list of accounts from the context. This allowed to have accounts propagated to overdue service.